pipeline.py

```diff
@@ -96,19 +97,27 @@ def sha(s):
             open(pickle_name, 'wb'),
             protocol=cPickle.HIGHEST_PROTOCOL)
 
+    # -- plot distributions:
+
     # # -- train
     # # design a Keras NN with three RNN streams (jets, photons, muons)
+    le=data['LabelEncoder']
+
+
     # # combine the outputs and process them through a bunch of FF layers
     # # use a validation split of 20%
     # # save out the weights to hdf5 and the model to yaml
-    net = train(data)
-
+    net=NN_train(data, model_name)
+   
     # # -- test
     # # evaluate performance on the test set
-    yhat = test(net, data)
-
+    yhat=net.predict([data['X_jet_test'], data['X_photon_test']], verbose = True, batch_size = 512) 
+  
     # # -- plot performance
+    #plot_NN(yhat, data)
+
     # # produce ROC curves to evaluate performance
+    save_roc_curves(yhat, data, le, model_name)
```

Comment:
minor detail: `le` is already included in `data`, so there is no need to extract it at line 104 and pass it to this function separately